pkgs/applications/finance/tradingview/default.nix

```
@@ -0,0 +1,92 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, squashfsTools
+, makeWrapper
+, alsa-lib
+, atk
+, at-spi2-atk
+, cups
+, gtk3
+, libdrm
+, libsecret
+, libxkbcommon
+, mesa
+, pango
+, sqlite
+, systemd
+, wayland
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tradingview";
+  version = "2.6.1";
+  revision = "44";
+  src = fetchurl {
+    url = "https://api.snapcraft.io/api/v1/snaps/download/nJdITJ6ZJxdvfu8Ch7n5kH5P99ClzBYV_${revision}.snap";
+    sha512 = "1ddd344d68cfb24774403ce9392c100ae330db49d6e27d71c51913d6b03de69cdb5edc3b5c5c6b25d3169335836ee72ea24536a91d9be6da2f007020c3d134b5";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
```

```suggestion
    makeBinaryWrapper
```
This is preffered, as can be seen in https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper